atd-vze/src/views/Users/UserForm.js

```diff
@@ -0,0 +1,289 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { Redirect } from "react-router-dom";
+import {
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  Col,
+  Form,
+  FormGroup,
+  FormText,
+  Input,
+  Label,
+  Row,
+  Alert,
+  Spinner,
+} from "reactstrap";
+
+const UserForm = ({ type, id = null }) => {
+  const token = window.localStorage.getItem("id_token");
+
+  const defaultFormData = {
+    name: "",
+    email: "",
+    password: "",
+    blocked: false, // Initialize blocked status
+    connection: "Username-Password-Authentication", // Set account type
+    verify_email: true, // Send email verification
+    app_metadata: {
+      roles: ["readOnly"], // Default to lowest level access
+    },
+  };
+
+  const [userFormData, setUserFormData] = useState(defaultFormData);
+  const [isFormDataLoaded, setIsFormDataLoaded] = useState(false);
+  const [isSubmissionError, setIsSubmissionError] = useState(false);
+  const [isFormSubmitted, setIsFormSubmitted] = useState(false);
+
+  const roles = [
```

Comment:
Yes, I finally found a way to make this happen in https://github.com/cityofaustin/atd-vz-data/pull/716 because it didn't feel right having these defined in more than one place.